monkey/monkey_island/deb-package/DEBIAN_MONGO/postinst

```diff
@@ -5,8 +5,8 @@ INSTALLATION_FOLDER=/var/monkey/monkey_island/installation
 PYTHON_FOLDER=/var/monkey/monkey_island/bin/python
 
 # Prepare python virtualenv
-pip2 install virtualenv --no-index --find-links file://$INSTALLATION_FOLDER
-virtualenv -p python2.7 ${PYTHON_FOLDER}
+pip3 install virtualenv --no-index --find-links file://$INSTALLATION_FOLDER
+virtualenv -p python3 ${PYTHON_FOLDER}
```

Comment:
also, should be 3.7, no?

Reply:
Then we break on Py3.8 and Py3.9, along with Py3.6 where we currently still work